lsfd-cmd/lsfd.1.adoc

```diff
@@ -2,7 +2,7 @@
 ////
 Copyright 2021 Red Hat, Inc.
 
-This file may be copied under the terms of the GNU Public License.
+This file may be copied under the terms of the GNU General Public License.
```

Comment:
I wonder where it came from.

Please consider correcting these files, too.
```
$ git ls-files | xargs grep -Hn 'GNU Public License' | grep -v po-man
disk-utils/fsck.8.adoc:4:This file may be copied under the terms of the GNU Public License.
disk-utils/swaplabel.8.adoc:4:This file may be copied under the terms of the GNU Public License.
disk-utils/swaplabel.c:11: * This file may be redistributed under the terms of the GNU Public License
include/pager.h:4: * This file may be redistributed under the terms of the GNU Public License.
libsmartcols/scols-filter.5.adoc:5:This file may be copied under the terms of the GNU Public License.
lsfd-cmd/lsfd.1.adoc:5:This file may be copied under the terms of the GNU Public License.
misc-utils/blkid.8.adoc:3:// This file may be copied under the terms of the GNU Public License.
misc-utils/fincore.1.adoc:5:This file may be copied under the terms of the GNU Public License.
misc-utils/findfs.8.adoc:3:// This file may be copied under the terms of the GNU Public License.
misc-utils/lastlog2.8.adoc:4:This file may be copied under the terms of the GNU Public License.
misc-utils/uuidd.8.adoc:4:This file may be copied under the terms of the GNU Public License.
misc-utils/uuidgen.1.adoc:4:This file may be copied under the terms of the GNU Public License.
misc-utils/wipefs.8.adoc:3:// This file may be copied under the terms of the GNU Public License.
sys-utils/eject.1.adoc:6:It may be distributed under the GNU Public License, version 2, or
```

Reply:
Nice! It's located in disk-utils/swaplabel.c since 2010 and has probably copied
and pasted it to many other files since then (let's blame the maintainer too ;-)).

Reply:
I'll fix it.